client/src/components/pages/admin/Submitted_Id/index.css

```diff
@@ -0,0 +1,30 @@
+.profile_body_table{
+  display: flex;
+  justify-content: space-around;
+}
+
+.profile__btn__toggle{
+  margin-top: 12px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.dashboard_page_admin{
+  width: 100vw;
+  height: 177vh;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  background-image: url('../../../../assets/images/dashbackground.png');
+  background-repeat: no-repeat;
+  background-size: 100vw 100vh;
+  background-attachment: fixed;
+}
+
+.scrollable_field{
+    width: 200px;
```

Comment:
@alidah0 It's not a big deal but the tabs look incorrect here, like these styes should not be tabbed over this far.